### PR TITLE
Rov 114 show server success and error message on file upload

### PR DESCRIPTION
**Pull request by Varuno8**

### CODE REVIEW

File: src/app/application/upload-statement/state/upload-statement.service.ts [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 export class UploadStatementService {
-    constructor(private httpClient: HttpClient) {
+    constructor(
+        private httpClient: HttpClient,
+        private toastr: ToastrService
+    ) { }
 
+    uploadStatement(formData: FormData) {
+        const apiUrl = "http://localhost:5053/api/FileUpload/upload"
+        return this.httpClient.post('apiUrl', formData)
```
Comment (rachittyagi) [inline src/app/application/upload-statement/state/upload-statement.service.ts:16]: the 'apiUrl'  should be without quotes

File: src/app/application/upload-statement/state/upload-statement.service.ts [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+            .pipe(
+                map((response: any) => { // The map operator takes the response and returns it directly.
+                    this.toastr.success('File uploaded successfully', 'Success');
```
Comment (rachittyagi) [inline src/app/application/upload-statement/state/upload-statement.service.ts:19]: the toastr should be at component level instead of service

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: src/app/application/upload-statement/state/upload-statement.service.ts (hunk 1 above)
Comment (Varuno8) [reply src/app/application/upload-statement/state/upload-statement.service.ts]: removed the quotes

Comment (Varuno8) [reply src/app/application/upload-statement/state/upload-statement.service.ts]: resolved the error

Comment (Varuno8) [reply src/app/application/upload-statement/state/upload-statement.service.ts]: removed the quotes

**Review by rachittyagi — APPROVED** [review]

**Pull request merged**